### DISCUSSION TITLE
[Feature] Removed general settings page (BREAKING CHANGE) (#1464)

### DISCUSSION THREAD

**Discussion by ilsaul** [alexjustesen/speedtest-tracker#1539] [Q&A]
Hi All,
more than Feature I feel it is a regression :-).
The fact that we can set the parameters at the time of installation is a good thing, but changing them while the application is running seems to me to be the basis of a good application otherwise what's the point of an administrator user?

Bye

**Comment by ilsaul** [comment]
I forget: if I want to test with a particular server one time what I need to do? 
1) change the configuration 
2) redeploy the application 
3) make the test
4) restore the configuration
5) redeploy the application

In my point of view is not usefull.

Bye

**Reply by OddSquirrel** [reply]
Granted, I feel the same, but @alexjustesen had his reasons. For some reason, having the settings inside the app turned out to be problematic in some cases.

You should be able to find instructions regarding the breaking changes [here](https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0).
